lib/mpl_toolkits/axes_grid1/mpl_axes.py

```diff
@@ -33,10 +33,12 @@ def __init__(self, axes):
 
         def __getitem__(self, k):
             if isinstance(k, tuple):
-                r = SimpleChainedObjects([dict.__getitem__(self, k1) for k1 in k])
+                r = SimpleChainedObjects(
+                    [super(Axes.AxisDict, self).__getitem__(self, k1)
```

Comment:
Couldn't I also just do `super(Axes.AxisDict, self)[k1]` here? dict subclassing is still a mystery to me